index.html

```diff
@@ -160,10 +160,24 @@ <h2>
       </ul>
       <p>
         The details of how to fulfill a payment request for a given <a>payment
-        method</a> are handled by <dfn data-lt="payment handler">payment
-        handlers</dfn>. In this specification, these details are left up to the
-        <a>user agent</a>, but future specifications may expand on the
-        processing model in more detail.
+        method</a> is an implementation detail of a <dfn data-lt=
+        "payment handlers">payment handler</dfn>. Additionally, each payment
+        handler defines a <dfn>can make payment</dfn> algorithm. The details of
+        how a payment handler determines wether it, or the user, can
+        potentially "make a payment" is also an implementation detail of a
+        payment handler. For an example, see the <a data-cite=
+        "payment-method-basic-card#can-makepayment">can make payment</a>
+        algorithm of [[payment-method-basic-card]].
+      </p>
+      <p>
+        A payment handler also defines the <dfn>steps to respond to a payment
+        request</dfn>, which results with an object, or <a data-cite=
```

Comment:
s/results with/returns

"object, or dictionary, that" -> "object or dictionary that"